hepdata/modules/converter/__init__.py

```diff
@@ -99,3 +91,19 @@ def convert_oldhepdata_to_yaml(input_path, output_path):
     )
 
     return successful
+
+
+@contextlib.contextmanager
+def prepare_data_folder(input_archive, input_format):
+    input_root_dir = tempfile.mkdtemp()
```

Comment:
Will do. 

Note that this was copied from the [previous code](https://github.com/HEPData/hepdata/blob/481b4aad7ec68c0289d95bdadab77dadbad8aa48/hepdata/modules/converter/__init__.py#L46) so the current version isn't using the correct temp directory.

Reply:
It was probably not done consistently in the previous code, but better to pass an explicit `dir` argument to all uses of `tempfile.mkdtemp` and `tempfile.TemporaryDirectory` throughout the code, otherwise the `CFG_TMPDIR` config option is not much use.  (I was looking in the wrong temp directory when debugging.)